src/mame/video/nmk16spr.h

```diff
@@ -0,0 +1,53 @@
+// license:BSD-3-Clause
+// copyright-holders:Mirko Buffoni,Nicola Salmoria,Bryan McPhail,David Haywood,R. Belmont,Alex Marshall,Angelo Salese,Luca Elia
+// thanks-to:Richard Bush
+#ifndef MAME_VIDEO_NMK16SPR_H
+#define MAME_VIDEO_NMK16SPR_H
+
+#pragma once
+
+#include "screen.h"
+
+typedef device_delegate<void (u32 &colour, u32 &pri_mask)> nmk16_spr_colpri_cb_delegate;
+typedef device_delegate<void (u16 attr, int &flipx, int &flipy, int &code)> nmk16_spr_ext_cb_delegate;
```

Comment:
Please move these delegates inside the public section of `nmk_16bit_sprite_device` and remove the name prefix (so their qualified names will be `nmk_16bit_sprite_device::colpri_delegate` and `nmk_16bit_sprite_device::spr_ext_delegate`).